app/slices/PartnersGrid/index.vue

```diff
@@ -46,6 +54,10 @@ defineProps(getSliceComponentProps<Content.PartnersGridSlice>(['slice', 'index',
 .lemon {
     fill: var(--pink-300);
 }
+.partners-intro {
+    position: relative;
+    z-index: 1;
+}
```

Comment:
Not sure if necessary.
Si tu met `z-index: -1` sur `BouncingAnimation` et `z-index: 0;` sur son parent ça devrait être en arrière de ce bloc